src/components/viewComment/Index.jsx

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import 'react-quill/dist/quill.snow.css';
+import {
+  Alert, Row, Col, Card
+} from 'react-bootstrap';
+import ReactHtmlParser from 'react-html-parser';
+import ReactQuill from 'react-quill';
+import Button from '../button/Index';
+import timeStamp from '../../utils/timeStamp';
+import { deleteComment, editComment } from '../../redux/actions/comments';
+import { getArticleRequest } from '../../redux/actions/articles';
+
+
+const ViewComment = (props) => {
+  const {
+    comment, time, id, slug, userId, user
+  } = props;
+
+  const date = timeStamp(time);
+  const [commentView, setCommentView] = useState({
+    commentBody: ''
+  });
+
+  const [errors, setErrors] = useState([]);
+
+  function renderErrors(errorsArray) {
+    return errorsArray.map(error => <Alert key={error} variant="danger">{error}</Alert>);
+  }
+
+
+  const onEdit = async (value) => {
+    const res = await props.editComment(slug, value, id);
+
+    if (res.status !== 205) {
+      setCommentView({ commentBody: '' });
+      return setErrors(['An error occured. Please try again later']);
+    }
+
+    if (res.status === 205) {
```

Comment:
@olajide1234 what is the probability that other status codes can be returned? What happens then?

Reply:
Line 37 of this file handles other status codes (`if (res.status !== 205)`). Since the feature is just for commenting, the most probable other status codes are server error codes which may not require the user to carry out any action but to try again.